universal-login-relayer/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "scripts": {
     "prebuild": "yarn clean",
-    "build": "tsc && tsc -p tsconfig.declaration.json",
+    "build": "tsc && rm build/lib/integration/sql/migrations/*.d.ts",
```

Comment:
Temporary solution until we update to newer version of `knex` that supports whitelisting extensions for migration files